types/v1beta1/firestore_client.d.ts

```diff
@@ -36,6 +36,7 @@ import * as protos from '../protos/firestore_v1beta1_proto_api';
  *  building truly serverless apps.
  * @class
  * @deprecated Use v1/firestore_client instead.
+ * @deprecated Use v1/firestore_client instead.
```

Comment:
Duplicated as well

Reply:
I wonder why it changes on types/v1beta1...
Is this the manual layer?

Reply:
I see the problem, the owlbot.py replace and copy to type
https://github.com/googleapis/nodejs-firestore/blob/main/owlbot.py#L144-L147
I think we should copy first and replace in type to avoid this issue.